custom-post-type/event-views/event-details.php

```diff
@@ -0,0 +1,118 @@
+<?php
+use mp_ssv_events\models\Event;
+use mp_ssv_events\models\Registration;
+use mp_ssv_events\SSV_Events;
+use mp_ssv_general\Form;
+use mp_ssv_general\Message;
+use mp_ssv_general\SSV_General;
+use mp_ssv_general\User;
+
+get_header();
+$event               = Event::getByID($post->ID);
+$event_registrations = $event->getRegistrations();
+?>
+<div id="page" class="container <?php echo is_admin_bar_showing() ? 'wpadminbar' : '' ?>">
+    <div class="row">
+        <div class="col s12 <?php echo is_dynamic_sidebar() ? 'm7 l8 xxl9' : '' ?>">
+            <div id="primary" class="content-area card">
+                <div class="card-image waves-effect waves-block waves-light">
+                    <?php if (has_post_thumbnail()): ?>
+                        <img src="<?php the_post_thumbnail_url() ?>"/>
+                    <?php else: ?>
+                        <div class="thumbnail-placeholder"></div>
+                    <?php endif ?>
+                    <div class="card-overlay hide-on-small">
+                        <div class="page-title">
+                            <h1><?= the_title() ?></h1>
+                            <h3><?= esc_html($event->getStart('F jS @ H:i')) ?> <?= $event->getEnd() != false && $event->getEnd() != $event->getStart() ? ' - ' . esc_html($event->getEnd('F jS @ H:i')) : '' ?></h3>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-content">
+                        <?php
+                        the_post();
+                        the_content();
+                        ?>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-content">
+                    <h3>Details</h3>
+                    <div class="row">
+                        <div class="col s12 xl6">
+                            <h5>Date and time</h5>
+                            <div class="row" style="border-left: solid; margin-left: 0; margin-right: 0;">
+                                <?php if ($event->getEnd() != false && $event->getEnd() != $event->getStart()): ?>
+                                    <div class="col s3">From:</div>
+                                    <div class="col s9"><?= esc_html($event->getStart()) ?></div>
+                                    <div class="col s3">Till:</div>
+                                    <div class="col s9"><?= esc_html($event->getEnd()) ?></div>
+                                <?php else : ?>
+                                    <div class="col s3">Start:</div>
+                                    <div class="col s9"><?= esc_html($event->getStart()) ?></div>
+                                <?php endif; ?>
+                            </div>
+                            <h5>Organiser</h5>
```

Comment:
I think it would look a bit messy to split the details block in to three separate blocks, since then the date/time and organizer blocks would be two very small blocks.

Reply:
I think you don't know how this works but lets see which of the three is the best:
https://develop.allterrain.nl/wp-content/uploads/2017/09/001-2.png
https://develop.allterrain.nl/wp-content/uploads/2017/09/002.png
https://develop.allterrain.nl/wp-content/uploads/2017/09/003.png
And most importantly why?

Reply:
Whoops, small misunderstanding, thought you meant we should put everything in separate cards. The blocks indeed should be split up differently. I think either the second or the third one is the best. If the location contains a embedded map, then it will take up a lot of vertical space and it wouldn't make sense to put it below organizer or date/time.

Reply:
And if there is no location, then organizer and date/time should be positioned next to each other.

Reply:
Here's the (same?) three choices, but then with a map for the location.

![screen shot 2017-09-18 at 20 50 22](https://user-images.githubusercontent.com/7968933/30558916-361822d8-9cb3-11e7-9a09-ce47582f162b.png)
&#x26;nbsp;
![screen shot 2017-09-18 at 20 51 10](https://user-images.githubusercontent.com/7968933/30558957-448bf4f2-9cb3-11e7-8127-6ef34b9bd3f7.png)
&#x26;nbsp;
![screen shot 2017-09-18 at 20 50 43](https://user-images.githubusercontent.com/7968933/30559016-87a33494-9cb3-11e7-8e13-3b2312cadef8.png)